_data/products/knight-point-systems.json

```diff
@@ -10,13 +10,13 @@
         "information technology and services",
         "inc 5000 2014 it services"
     ],
-    "short_description": "Knight Point Systems’ Cloudseed is a portfolio of solutions that spans a customer’s technology needs across multiple phases of cloud adoption.",
-    "long_description": "Knight Point Systems’ Cloudseed is a portfolio of solutions that spans a customer’s technology needs across multiple phases of cloud adoption.  The KPS Federal Community Cloud is an off-premise IaaS solution that leverages Cisco hardware, world-class Equinix facilities, and open-source technology to provide compute, storage, and network resources to Federal customers in a secure manner.  Customers can engage resources in an a la carte, and “pay for performance” model that allows them to know exactly what to expect – It’s the technology you need, when you need it.  The resources are managed and monitored through the KPS tool “Zeus”, which allows management of cloud resources across all of a customer’s cloud deployments – Public, Community, and Private – giving them the ability to have true “hybrid” cloud solutions that are right-sized for their needs.",
-    "sales_poc":"mash@knightpoint.com",
+    "short_description": "The KPS Federal Community Cloud, called CloudSeed, is an off-premises IaaS solution that leverages Cisco hardware, world-class Equinix facilities, and open-source technology to provide compute, storage, and network resources to Federal customers in a secure manner."
```

Comment:
missing a comma at the end of the "short_description" which will cause this file not to compile